README.md

```diff
@@ -27,16 +27,40 @@ Or install it yourself as:
       environment: "production",
       application: "ror"
     }
-    logger = Hivelog::Logger.new(:stdout, @labels)
+
     options = {
       event: event,
       user: user,
       group: group,
       tags: tags
     }
-    logger.debug("deb ya", options)
+
+    Hivelogger = Hivelog::Logger.new(:stdout, @labels)
+
+    Hivelogger.debug("deb ya", options)
+    Hivelogger.warn("warning message", options)
+    Hivelogger.info("information message", options)
+    Hivelogger.error("error message", options)
 ```
 
+Each log is a sentence. A sentence always starts with capitalization and ends with a period.
+
+Please scope the service in `service`, the module in `module` and the scope with `tags` (e.g.: `donations`, `payment`, `recurly`, ...). It will help for filtering.
+
+```ruby
+    options = {
+      service: 'webapp',
+      module: 'topics',
```

Comment:
`Service` + `module` type password/text are custom field, i don't suggest to use here. Please check ECS doc.